package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mediawiki-title",
-  "version": "0.7.0",
+  "version": "0.8.0",
```

Comment:
Since this is mostly a refactor, I don't think we need to bump the version, especially not the minor component.

Reply:
Done